pytorch_lightning/profiler/profilers.py

```diff
@@ -145,30 +220,32 @@ class SimpleProfiler(BaseProfiler):
     the mean duration of each action and the total time spent over the entire training run.
     """
 
-    def __init__(self, output_filename: Optional[str] = None, extended=True):
+    def __init__(
+        self,
+        dirpath: Optional[Union[str, Path]] = None,
```

Comment:
dirpath and filename is not backward compatible. Any idea to better handle this ?

Reply:
we still have `output_filename` for BC